server/cla.go

```diff
@@ -121,3 +130,15 @@ func checkCLAComment(comments []*github.IssueComment, username string) (int64, b
 	}
 	return 0, false
 }
+
+func isUserExcluded(exclusions []string, username string) bool {
```

Comment:
Maybe a more generic name like `contains` would fit the method better, as it's not specific for users.

Reply:
+1